### PR TITLE
[MDB IGNORE] Blueshift access fixes

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

makes some nuimber in to word

## How This Contributes To The Skyrat Roleplay Experience

I don't know I hate mapping I just want the problems to go away

## Changelog
:cl:
fix: Fixed some airlock access issues on Blueshift
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
oops

**Comment by YakumoChen** [comment]
isn't this what we have access helpers for now?

**Comment by tf-4** [comment]
sounds like a mapper issue

**Comment by YakumoChen** [comment]
they are map objects you add to doors that give the door access instead of having to varedit them

File: _maps/map_files/Blueshift/BlueShift_lower.dmm [hunk 1]
```diff
@@ -12398,7 +12398,7 @@
 /area/station/maintenance/starboard)
 "jRk" = (
 /obj/machinery/door/airlock/maintenance_hatch{
-	req_one_access_txt = "12;47"
+	req_one_access_txt = list("maint_tunnels","research")
```
Comment (Jolly-66) [inline _maps/map_files/Blueshift/BlueShift_lower.dmm:12401]: You make me want to gauge your eyes out

Comment (tf-4) [reply _maps/map_files/Blueshift/BlueShift_lower.dmm:12401]: :)

**Pull request closed without merging**